Assets/BossRoom/Scripts/Editor/BakingMenu.cs

```diff
@@ -0,0 +1,148 @@
+        {
+            Debug.Log("No light probes found in scene");
+        }
+
+        else
+        {
+            foreach (var index in lightProbes)
+            {
+                index.GetComponent<LightProbeGroup>().enabled = true;
+                Debug.Log("Light probes enabled");
+            }
+        }
+    }
+    
+    [MenuItem("Boss Room/Lighting Setup/Light Probes/Disabled")]
+    static void LightProbesDisabled()
+    {
+        GameObject[] lightProbes = GameObject.FindGameObjectsWithTag("LightingProbes");
+
+        if (lightProbes == null || lightProbes.Length == 0)
+        {
+            Debug.Log("No light probes found in scene");
+        }
+
+        else
+        {
+            foreach (var index in lightProbes)
+            {
+                index.GetComponent<LightProbeGroup>().enabled = false;
+                Debug.Log("Light probes disabled");
+            }
+        }
+    }
+    
+    [MenuItem("Boss Room/Lighting Setup/Reflection Probes/Baked")]
+    static void ReflectionProbesBaked()
+    {
+        GameObject[] reflectionProbes = GameObject.FindGameObjectsWithTag("LightingReflectionProbe");
+
+        if (reflectionProbes == null || reflectionProbes.Length == 0)
+        {
+            Debug.Log("No reflection probes found in scene");
+        }
+
+        else
+        {
+            foreach (var index in reflectionProbes)
+            {
+                index.GetComponent<ReflectionProbe>().mode = ReflectionProbeMode.Baked;
+                Debug.Log("Reflection probes set to Baked");
+            }  
+        }
+    }
+    
+    [MenuItem("Boss Room/Lighting Setup/Reflection Probes/Realtime")]
+    static void ReflectionProbesRealtime()
+    {
+        GameObject[] reflectionProbes = GameObject.FindGameObjectsWithTag("LightingReflectionProbe");
+        
+        if (reflectionProbes == null || reflectionProbes.Length == 0)
+        {
+            Debug.Log("No reflection probes found in scene");
+        }
+
+        else
+        {
+            foreach (var index in reflectionProbes)
+            {
+                index.GetComponent<ReflectionProbe>().mode = ReflectionProbeMode.Realtime;
+                Debug.Log("Reflection probes set to Realtime");
+            }  
+        }
+    }
+    
+    [MenuItem("Boss Room/Lighting Setup/All Baked")]
+    static void AllBaked()
+    {
+        EnvironmentLightsBaked();
+        ReflectionProbesBaked();
+        Debug.Log("All lights and reflection probes in scene set to Baked");
+    }
+    
+    [MenuItem("Boss Room/Lighting Setup/All Realtime (except area lights)")]
+    static void AllRealtime()
+    {
+        EnvironmentLightsRealtime();
+        ReflectionProbesRealtime();
+        Debug.Log("All lights and reflection probes in scene set to Realtime");
+    }
+    
+    [MenuItem("Boss Room/Lighting Setup/Clear ALL Baked Data")]
+    static void ClearBakedData()
+    {
+        Lightmapping.Clear();
```

Comment:
I noticed this doesn't delete the lightmaps from your project, only the references to lightmaps are unlinked. 

Shouldn't the lightmap itself be deleted? Once the reference to the lightmap is removed, you're not able to re-link lightmaps through the editor, so you'd have to re-bake anyways, correct?

Reply:
Resolving this as the consensus offline was to direct users where to clear and generate bake data.